src/csvToJson.js

```diff
@@ -56,8 +50,10 @@ const assign = (file, dataEntries) => {
 // if it's main - then let's put it into index.js
 // @TODO when we'll have getHeaders method working, should we call it inside of this method?
 // @TODO can we make it better? should we have all of those attributes?
-const csvToJson = async (path, fileInfo, headers) => {
-  const data = await parseCsv(path);
+const csvToJson = async (path, headers) => {
+  const fullPath = ''.concat(path[0], path[1], '.csv');
+  const data = await parseCsv(fullPath);
```

Comment:
There is an issue I think.
we passing headers into csvToJson, but don't pass them into parseCsv.
We talk about it with @EbrahimKreem but it was not resolved yet

can you also elaborate `assign` call? I just don`t remember what this method is doing

Reply:
Ok. 
assign chunks the array of data entries into json files with 10000 entries each
so array with 82000 entries chunks into 9 files. Each file needs a diff name, so we add 0,1,2,3,4,5,6,7,8 to end of each file name. Then file is generated. 

The headers part is confusing, as I haven't worked on that part. But given properly formatted data, the write is working

Reply:
about assign:
Ok, I think we need to test a new version and then my worries will be solved. 
Some csv files don't have huge datasets, so it will crucial to test how it will work. But good call.

About `ParseCsv` - check the place where this method located and also explore how parser using it.
and compare with line at csvToJson.

Reply:
Btw, maybe assign actually replacing `write` method? i mean with `write` we creating a new file, right? and `assign` will create files with same data as well?

Reply:
assign doesn't create the files, it just makes the filename + the file number, then passes to `generate`, which uses `write` to make the file. We could just get put `generate` and `assign` in 1 actually, but it would be a longer function.

Reply:
but inside of assign it still uses `write`, so i'm correct that we cannot use both methods inside of csvToJson.

---

i think you can check how fao works without my input :)
try it on your side and then show me a screenshot - i'm a bit busy trying to figure out how to fix `generator` bug

---

> We could just get put generate and assign in 1 function actually, but it would be a longer function.

I dont like long methods.

Reply:
Does write partition the data into pieces?

Reply:
I don't think so. main aim for `write` was to write data into the file.
and functionality, related to `split` was introduced quite later, when we add 2 more datasets and we realize that they are pretty large